src/store.js

```diff
@@ -0,0 +1,23 @@
+
+import {
+  createStore, applyMiddleware, combineReducers, compose,
+} from 'redux';
+import movieReducer from 'Components/reducers';
+import createSagaMiddleware from 'redux-saga';
+import mySaga from 'Components/saga';
+
+
+const allReducers = combineReducers({
```

Comment:
please create new file where you combine all reducers

Reply:
basically you should have App folder that includes all related staff: App definition, store, all reducers etc (it's described in the article about domain approach I've sent before)